map.go

```diff
@@ -184,12 +190,17 @@ func createMap(spec *MapSpec, inner *internal.FD, handle *btf.Handle) (*Map, err
 		}
 	}
 
+	if spec.NumaNode > 0 {
```

Comment:
Looking at the kernel source, I think that node 0 is actually valid. So I think we can't "automagically" set this flag.

Reply:
In most cases the user does not need to set a NUMA node, even if there are multiple sockets that will be assigned to node 0 if not specified.

Anyway, we can remove it if you don't feel up to it.